### PR TITLE
Dynamic branching for CONUS

### PR DESCRIPTION
**Pull request by wdwatkins**
To really optimize this it might make sense to also branch the `oldest_active_sites` target, so that it doesn't re-inventory every state when one is added/removed.  I didn't miss this somewhere did I?

### CODE REVIEW

File: _targets.R [hunk 1]
```diff
@@ -19,25 +21,14 @@ source('3_visualize/src/plot_data_coverage.R')
...
-             plot_site_data(state_plot_files, nwis_data, parameter),
-             format = 'file'),
-  names = state_abb,
-  unlist = FALSE
-)
-
-
 list(
   # Identify oldest sites
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
```
Comment (lindsayplatt) [inline _targets.R:34]: I think you may have missed the step where we fix the part where it re-inventories all when just one state changes. Head back to the 3rd task at https://github.com/wdwatkins/ds-pipelines-targets-3/issues/12#issuecomment-904868710

Comment (lindsayplatt) [reply _targets.R:34]: Oh wait ... nevermind. I was up a target to high. Disregard.

**Review by lindsayplatt — APPROVED** [review]
This implements what I expected. You're right - we could have reworked the `find_oldest_sites()` function to be called per state instead (I think this is how I would go about implementing from scratch), but we wanted to give an example of using splitters. See more / contribute your thoughts to this discussion about splitters and how best to introduce them: https://github.com/USGS-R/ds-pipelines-targets-3-course/issues/8

**Comment by wdwatkins** [comment]
Cool, just wanted to be sure.

**Pull request merged**